components/Form/Form.jsx

```diff
@@ -192,51 +227,45 @@ const Form = (props) => {
 
                         <FormInput
                             name='linkedin_profile'
+                            type='url'
                             fieldStyle={styles.linkedinField}
-                            labelContent={fields.linkedin.label}
-                            placeholder={fields.linkedin.placeholder}
-                            value={formValues.linkedinProfileLink}
+                            labelContent={getField(fields, 'linkedin').label}
+                            placeholder={getField(fields, 'linkedin').placeholder}
+                            value={formValues.linkedin_profile}
                             onChange={handleChange}
                         />
 
                         <FormInput
                             name='email'
-                            mandatory='yes'
+                            type='email'
+                            required='required'
```

Comment:
It's not working. We can still send the form without fill this field.

Reply:
ontem estava funcionando... o q nao funciona?

Reply:
acabei de rodar aqui e esta validando